### DISCUSSION TITLE
NGINX Reverse proxy

### DISCUSSION THREAD

**Discussion by deg749Goroe3** [cupcakearmy/cryptgeon#61] [Q&A] [answered]
Hello, sorry for the noob question.

I have web server on Linux with nginx running. How can I run cryptgeon and reverse this app without another nginx in container? Because in this case nginx on system stop working.

**Comment by cupcakearmy** [comment, marked as answer]
Hey there :)

Could you maybe post you nginx config? I haven't touched nginx in a while but maybe I can help.

In theory what you would do is run the service on an internal port (e.g. 1234) different from your current port and them proxy it thorugh nginx with a subdomain.

**Reply by deg749Goroe3** [reply]
I'm using default nginx config from your examples. 
Can you please guide me how can I run cryptgeon with a different port, so I can run two instances of nginx server?

**Reply by cupcakearmy** [reply]
It's right here

https://github.com/cupcakearmy/cryptgeon/blob/c40f00952371a60f41d0ecc7fb3f5e4902aae4ae/examples/nginx/nginx-plain.conf#L2-L3

**Comment by deg749Goroe3** [comment]
Thank you. All works now. Appreciate this.
